deployment/k8s/templates/common-env.yaml

```diff
@@ -338,6 +338,13 @@ Service-specific environment variables that utilize the standardized variables
   value: {{ int .Values.global.maxPovSize | quote }}
 {{- end }}
 
+{{- define "buttercup.fuzzerRunnerEnv" }}
+- name: BUTTERCUP_FUZZER_RUNNER_HOST
+  value: "0.0.0.0"
```

Comment:
I forget exactly how it works with the sidecar containers now, but assuming we end up with two fuzzers on the same node, will the runner-proxy be able to figure out which fuzzer-runner instance to reach? I assume it works implicitly on a single node-single fuzzer instance deployment.